src/libraries/Common/tests/System/Security/Cryptography/MLKemTests.cs

```diff
@@ -453,7 +469,14 @@ public static void ImportPkcs8PrivateKey_Both_TrailingData()
             }
         }
 
-       [Fact]
+        [Fact]
+        public static void ImportPkcs8PrivateKey_NotAsn()
+        {
+            Assert.Throws<CryptographicException>(() => MLKem.ImportPkcs8PrivateKey("potatoes"u8));
+            Assert.Throws<CryptographicException>(() => MLKem.ImportPkcs8PrivateKey("potatoes"u8.ToArray()));
```

Comment:
I don't see a test of "trim off the last byte, might require 768 or 512 because of preconditions", but I guess it's "make ReadEncodedValue throw", which this does.

Do these actually trigger that?  Or do they hit a minimum size precondition?

Reply:
There is no minimum size pre-condition for PKCS#8 imports on ML-KEM, only on export destination buffers. We can add one, but it seemed a bit unnecessary, and we don't do it on existing algorithms.